spec/expression.dd

```diff
@@ -4,11 +4,31 @@ $(SPEC_S Expressions,
 
 $(HEADERNAV_TOC)
 
+$(P This chapter defines the syntax, order of evaluation, and semantics of expressions. An expression is a sequence
+of operators and operands that specifies an evaluation.)
+
     $(P Expressions are used to compute values with a resulting type.
         These values can then be assigned,
         tested, or ignored. Expressions can also have side effects.
     )
 
+$(H2 $(LNAME2 definitions-and-terms, Definitions and Terms))
+
+$(DDOC_ANCHOR define-full-expression)$(P $(B Definition) ($(DOUBLEQUOTE Full expression)): For any expression
+$(I expr) in a D program, we define the full expression of $(I expr) as follows. If $(I expr) parses as a
```

Comment:
s/we define the full expression of $(I expr)/the full expression of $(I expr) is defined/

On a formal document, "we" should not be there.